step-function/get-files/src/main.py

```diff
@@ -12,7 +12,7 @@ def get_download_url(bucket, key):
     return f'https://{bucket}.s3.{region}.amazonaws.com/{key}'
 
 
-def get_expiration_time(bucket, key):
+def get_expiration_time(key, bucket):
```

Comment:
I'd vote for reverting this, I'm used to seeing/thinking `bucket, key` from all the boto3 functions